src/solvers/dg.jl

```diff
@@ -214,6 +219,12 @@ function Base.show(io::IO, mime::MIME"text/plain",
     end
 end
 
+function get_node_variables!(node_variables, mesh, equations,
+                             volume_integral::VolumeIntegralSubcellLimiting, dg, cache)
+    get_node_variables!(node_variables, volume_integral.limiter, volume_integral,
```

Comment:
Do you need to update the indicator values here again just like in https://github.com/trixi-framework/Trixi.jl/pull/1611/files#diff-2c7d36606eb2ae74d8571195c5adc60bbdbd969e6fea398fc7ebaaf24d64a203R140?

Reply:
Honestly, I'm not quite sure, why you update the element coefficients before saving. Is it to save the alphas which are used for the current solution and therefore for the next limitation step?
Because right now I'm just saving the last used coefficients. (By the way, this is why there are none for the first solution file.)

Reply:
> Is it to save the alphas which are used for the current solution and therefore for the next limitation step?

Yes. We wanted to have a consistent set of values in the solution files, such that it is not "solution at t = t^n" and "indicator at t = t^{n-1}". But if this is not an option for the your limiter (maybe because a re-evaluation would be too expensive or otherwise problematic), I think it is OK to just save the previously used values.

Or would you disagree here, @ranocha? I don't have a strong opinion...

Reply:
I think it would be helpful to keep the consistency we have right now. Otherwise, I fear it may result in hard-to-debug issues... In my opinion, "too expensive" does not really count since IO is already very expensive.

Reply:
> In my opinion, "too expensive" does not really count since IO is already very expensive.

This is also a good point 👍

Reply:
I just tried to implement this updating step for the limiter coefficients. The problem is that the calculation of the coefficients is done with the stage callback `SubcellLimiterIDPCorrection. This and the associated function cannot be accessed except as this callback. So, I would have to hard code this part like 
```
SubcellLimiterIDPCorrection()(u_ode, semi, t, dt, volume_integral)
get_node_variables!(node_variables, volume_integral.limiter, volume_integral,
                    equations)
```
Moreover, in the Correction function, the solution is directly corrected and change (obviously with that name). Of course, I could add the functionality to do the correction only if wanted.

In fact, this would be a more serious problem for the MCL, since the coefficient (in [0,1]) is only calculated for visualization afterward, but mainly the antidiffusive flux is limited directly in `calcflux_antidiffusive_limited!`. Preventing a double limitation here would probably not be so nice.

Overall, I wanted to ask if you think this is worth implementing. @ranocha @sloede

Reply:
@sloede I think we agreed that all this part will be considered less polished/maintained than the standard part of Trixi.jl, didn't we? In this case, I don't have any strong opinion - except adding comments explaining this.

Reply:
Yes, I agree. It would be good to leave a short comment explaining that updating the indicator is not possible and then add a link to @bennibolm's comment above (https://github.com/trixi-framework/Trixi.jl/pull/1611#discussion_r1334553206).

Reply:
Okay, perfect. I will add the comment including the reference and then request another review of you guys. Thank you.